### ISSUE TITLE
Audit: Eligibility for VA disability benefits

### ISSUE THREAD

**Issue by DanielleThierryUSDSVA** [department-of-veterans-affairs/va.gov-team#53815]

**Comment by DanielleThierryUSDSVA** [comment]
@mnorthuis Beth and I did a test of auditing against a paired down list of content things to check for in the audit. While we were reviewing this page, we checked the IA things that I had added to the audit file. I'm flagging this for you and I to sync on in our next sync!

**Comment by mnorthuis** [comment]
@DanielleThierryUSDSVA can we sync on this one?  I want to sync on what is being audited and have the IAs participate on them.  I'd say side nav label is also incorrect here.  Also curious how far the scope goes, because technically the breadcrumb should be "Home > VA disability compensation > Eligibility for VA disability benefits based on the hub page H1, so wondering if this is our opportunity to correct pages.

**Comment by DanielleThierryUSDSVA** [comment]
Yes, of course @mnorthuis! These first few are essentially Beth and I test-driving this process.
